src/Feedback.tsx

```diff
@@ -1,6 +1,15 @@
 import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
+import { BsPrefixProps, BsPrefixRefForwardingComponent } from './helpers';
+
+export interface FeedbackProps extends BsPrefixProps {
+  className?: string;
+  bsPrefix?: never;
+  type?: 'valid' | 'invalid';
+}
+
+type Feedback = BsPrefixRefForwardingComponent<'div', FeedbackProps>;
```

Comment:
**nitpick:** it's a little confusing, maybe, to have two things e.g. both named `Feedback` here? maybe we can just inline the type def?